### ISSUE TITLE
the dev_compiler's CLI has a lot of command-line options

### ISSUE THREAD

**Issue by devoncarew** [dart-lang/dev_compiler#361]
`dev_compiler -h` is 68 lines long. It's a lot to process - fewer options would be better. Some suggestions are to remove no longer relevant options, perhaps simplify existing options, and splitting some of the work into subcommands, so the main options page isn't as large (ddc analyze; ddc compile; ddc serve, ...)

**Comment by jmesserly** [comment]
Already have a CL out that removes all of strong mode options :)

Other than that:
* the whole server feature needs to go, that will kill most of them
* DDC analyze won't exist, use dartanalyzer --strong
* => bind workaround option is almost dead, once we can move from io.js to node4 (there's a pull request, last I checked we were blocked on one of Karma's dependencies not supporting that node version yet)

that will leave a handful of options that are actually useful for codegen. :tada:

**Comment by jmesserly** [comment]
I think this issue is obsolete at this point
